tests/query.test.js

```diff
@@ -65,4 +81,8 @@ describe('Query Array', () => {
   it('can filter a regular array', () => {
     expect([...filter(array, Boolean)]).toEqual([true, true]);
   });
+
+  it('can find a regular array', ()=>{
```

Comment:
I'd clarify here since the array isn't what's found, it's what's _in_ the array.

```js
it('can find specific items inside a normal array')
```